datajoint/dependencies.py

```diff
@@ -100,5 +105,9 @@ def descendants(self, full_table_name):
         :param full_table_name:  In form `schema`.`table_name`
         :return: all dependent tables sorted in topological order.  Self is included.
         """
-        nodes = nx.algorithms.dag.descendants(self, full_table_name)
-        return [full_table_name] + nx.algorithms.dag.topological_sort(self, nodes)
+
+        nodes = self.subgraph(
```

Comment:
I think it would make sense to just use `nx.DiGraph(self).subgraph(...)` here instead just like we do in the `ERD` and don't make `connection` optional. Also, the variable name `nodes` is a bit misleading - I'd say give it a name that suggests that it is a graph and not a mere list of nodes.